build.gradle

```diff
@@ -1,6 +1,28 @@
-group 'multi-int-store'
-version '0.1.0-SNAPSHOT'
+plugins {
+	id 'org.springframework.boot' version '2.1.4.RELEASE'
+	id 'java'
+}
+
+apply plugin: 'io.spring.dependency-management'
+
+group = 'com.connexta'
+version = '0.0.1-SNAPSHOT'
```

Comment:
I think we should revert the group back to `group 'multi-int-store'`. This what the Replication and Transformation repos do.